### PR TITLE
Adobrev/android key

### PR DESCRIPTION
**Pull request by AntonDobrev**

### CODE REVIEW

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Push Notifications to Android stopped working
```
Comment (Mike4o) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: Use title case:
"Push Notifications to Android Stopped Working"

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+description: Check your Google API key if you receive the error "Error authenticating to Google Cloud Messaging"
```
Comment (Mike4o) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: Describe only the error, not the solution. E.g.:
Push notifications throw "Error authenticating to Google Cloud Messaging"

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications, Android</td>	
+  </tr>
+</table>
+
+## Description
+
+Your Android app may have stopped receiving push notifications even they have been working for a significant amount of time. [Checking the reason for failing push notifications](http://docs.telerik.com/platform/backend-services/javascript/push-notifications/troubleshooting/push-trb-all.html#q-how-do-i-find-out-what-is-the-reason-for-not-receiving-push-notifications) in {{site.tp}} reveals the following error message.
```
Comment (Mike4o) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: I would rather say this to keep the customer context:
"Android app has suddenly stopped receiving push notifications due to the following error: `Error authenticating to Google Cloud Messaging server.`"

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Check the type of your Android API key:
+
+1. Open the [Google Developers console](https://console.developers.google.com/apis/library)
+1. From the projects list select your project - match its number to the number used to subscribe the mobile app for push notifications (as `senderId`). 
+1. On the left, click **Credentials**
+1. In the **API keys** list compare the Google API key used in {{site.tp}}
+1. Check if this key is of type **Server**. Other key types like *Android*, *Browser*, *iOS* are not supported. If the latter, you need to issue a **Server** API key and enter it in {{site.tp}}. 
+
+Issue a **Server** API key
```
Comment (Mike4o) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: The whole Issue a Server API key area shouldn't be here. It should be in a separate KB or linked externally. If you think that its place is in this KB, move it in the Notes section!

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Your Android app may have stopped receiving push notifications even they have been working for a significant amount of time. [Checking the reason for failing push notifications](http://docs.telerik.com/platform/backend-services/javascript/push-notifications/troubleshooting/push-trb-all.html#q-how-do-i-find-out-what-is-the-reason-for-not-receiving-push-notifications) in {{site.tp}} reveals the following error message. 
+
+## Error Message
+
+`Error authenticating to Google Cloud Messaging server.`
+
+## Cause
+
+The error signifies that {{site.tp}} cannot connect to Google Cloud Messaging (GCM) which is the vendor service responsible for delivering push notifications to the Android devices registered in your {{site.tp}} app. Due to recent changes in the GCM infrastructure this is the only type of key supported for server-to-server  communication. In {{site.tp}} you may have entered a Google API key of a type which is no longer supported by Google for authentication to GCM.
```
Comment (Mike4o) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: Fix this as discussed on Lync.

File: troubleshooting/troubleshooting-android-push-notifications-stopped-working.md (hunk 3 above)
Comment (AntonDobrev) [inline troubleshooting/troubleshooting-android-push-notifications-stopped-working.md]: I want to urge the customer to check the reason explicitly

**Pull request merged**